src/index.ts

```diff
@@ -39,12 +39,18 @@ if (!database) {
   process.exit(5);
 }
 
+const intervalId = setInterval(async () => {
+  const currentTimestamp = new Date().getTime();
+  console.log("Cleaning orders that have an expired date inferior to", currentTimestamp);
+  await database.deleteExpiredOrder(currentTimestamp);
```

Comment:
```suggestion
  database.deleteExpiredOrder(currentTimestamp / 1000);
```

Reply:
I cleaned this part, why use seconds since JS use milliseconds ? Is it linked with a usage on the ui side ?

Reply:
As Don said, contract uses seconds.